check_includes.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -f
+IFS='
+'
+GREEN="\033[0;32m"
+BLUE="\033[0;34m"
+PURPLE="\033[0;35m"
+NC="\033[0m"
+errors=0
+
+last_filename=
+last_line=
+last_import=
+last_import_line=
+last_import_class=
+for f in $(grep -rwHn 'import' wl)
+do
+  filename="${f%%:*}"
+  line="${f#*:}"
+  line="${line%%:*}"
+  file_and_line="$PURPLE$filename$NC:$BLUE$line$NC"
+
+  if [ "$filename" = "$last_filename" ]
+  then
+    if [[ "$last_line"+1 -ne "$line" ]]
+    then
+      echo -e "Gap in $PURPLE$filename$NC between import lines $BLUE$last_line$NC and $BLUE$line$NC"
+      ((errors++))
+    fi
+  fi
```

Comment:
This is resolved by the auto-format
(see https://github.com/widelands/wl_addons_server/pull/102/commits/cf7fa8d1b678e4f957eb78016146431dd4d78976)

Reply:
yes but clang-format sometimes changes its rules in a new version. No harm in having this check anyway, IMHO

Reply:
Then you should change the order of the steps in `check_lists.yaml`.
Otherwise clang-format doesn't get a chance to remove empty lines.